src/main/java/com/org/jenkins/custom/jenkins/distribution/service/GithubController.java

```diff
@@ -0,0 +1,35 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import java.util.List;
+import org.kohsuke.github.GHApp;
+import org.kohsuke.github.GHAppInstallation;
+import org.kohsuke.github.GHAppInstallationToken;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import static com.org.jenkins.custom.jenkins.distribution.service.GithubJwtHelper.createJWT;
+
+public class GithubController {
+
+    private static String APP_IDENTIFIER="";
```

Comment:
Should I load this via environment variables ?

Reply:
Yes, it's always good to pull things into environment variables

Reply:
I put it in the application.properties, is that enough ?